File: corporate-reporting/src/types/FinancialRiskMitigationActionUpdate.ts
import { IRiskMitigationActionUpdate, RiskMitigationActionUpdate } from "./RiskMitigationActionUpdate";
import { IFinancialRiskMitigationAction } from "./FinancialRiskMitigationAction";

export interface IFinancialRiskMitigationActionUpdate extends IRiskMitigationActionUpdate {
    FinancialRiskMitigationAction?: IFinancialRiskMitigationAction;
}

export class FinancialRiskMitigationActionUpdate extends RiskMitigationActionUpdate implements IFinancialRiskMitigationActionUpdate { }

File: ORB.Core/Models/FinancialRiskMitigationActionUpdate.cs
using System;
using System.Collections.Generic;

namespace ORB.Core.Models
{
    public partial class FinancialRiskMitigationActionUpdate : RiskMitigationActionUpdate
    {
        public virtual FinancialRiskMitigationAction FinancialRiskMitigationAction { get; set; }
        public virtual FinancialRiskUpdate FinancialRiskUpdate { get; set; }
    }
}
